hybrid/ccloud-integration/confluent-platform.yaml

```diff
@@ -105,3 +105,21 @@ spec:
       url:  https://connect.confluent.svc.cluster.local:8083
       tls:
         enabled: true
+---
+apiVersion: platform.confluent.io/v1beta1
+kind: KafkaRestProxy
+metadata:
+  name: kafkarestproxy
+  namespace: confluent
+spec:
+  dependencies:
+    schemaRegistry:
```

Comment:
shouldn't there be a kafka dependencies as well ?

Reply:
nice catch thanks!